### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by frankyston**
This Pull Request is to make it easier for new contributors to install the system on localhost with the login social network keys.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +103,76 @@ $ docker-compose up
...
+
+Fill in all fields like app name, description, website URL. In `Callback URLs` fill in with` http: // localhost: 3000 / auth / twitter / callback`. In `Allow this application to be used to sign in with Twitter` enable` Enable Sign in with Twitter`.
+
+Save and after saving go to the `Keys and tokens` tab to access the keys.
+
+## Accessing Google Variables
+
+Access the https://code.google.com/apis/console/
+
+Clique on the `Criar projeto` button.
```
Comment (lucianosousa) [inline README.md]: missing translation?

Comment (frankyston) [reply README.md]: @lucianosousa 474666550d7e319b70cc4593b339b9e7ec915772

File: README.md [hunk 2]
```diff
@@ -101,6 +103,76 @@ $ docker-compose up
...
+Click on the `Add new app` button.
+
+Fill in the name of the app and your contact email. Then click Create Application ID.
+
+On your app page, click on my left side under `Settings -> Basic`.
+
+In this page you need to configure two fields:
+
+- Application domain with `localhost`
+- In the site session the site URL field with `http: // localhost: 3000`, where 3000 is the port used to start the Rails application on your computer.
```
Comment (lucianosousa) [inline README.md]: I would prefer if instead of `http: // localhost: 3000` we use `http://localhost:3000` format

Comment (frankyston) [reply README.md]: @lucianosousa c52904b8aa89b15d888a90bee0d10a835ba77661

**Pull request merged**